tests/integration/api/v1/email-confirmation/patch.test.js

```diff
@@ -304,23 +306,13 @@ describe('PATCH /api/v1/email-confirmation', () => {
         }),
       });
 
-      const responseBody = await response.json();
-
-      expect(response.status).toEqual(400);
-
-      expect(responseBody).toStrictEqual({
-        name: 'ValidationError',
-        message: 'O email informado já está sendo usado.',
-        action: 'Ajuste os dados enviados e tente novamente.',
-        status_code: 400,
-        error_id: responseBody.error_id,
-        request_id: responseBody.request_id,
-        error_location_code: 'MODEL:USER:VALIDATE_UNIQUE_EMAIL:ALREADY_EXISTS',
-        key: 'email',
-      });
+      expect(response.status).toBe(200);
 
       const userInDatabaseCheck1 = await user.findOneById(firstUser.id);
-      expect(userInDatabaseCheck1.email).toEqual('validation.error@before.com');
+      expect(userInDatabaseCheck1.email).toBe(firstUser.email);
```

Comment:
Acho que a versão antiga era mais garantida:

```suggestion
      expect(userInDatabaseCheck1.email).toBe('validation.error@before.com');
```